I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 15, in script
    scene movie with black
  File "game/script.rpy", line 15, in <module>
    scene movie with black
NameError: name 'black' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 15, in script
    scene movie with black
  File "renpy/ast.py", line 1439, in execute
    paired = renpy.python.py_eval(self.paired)
  File "renpy/python.py", line 2276, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "renpy/python.py", line 2269, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/script.rpy", line 15, in <module>
    scene movie with black
NameError: name 'black' is not defined

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
Огонёк 
Sun Jun 26 01:06:57 2022
